framework/sdk/client.mdx

```diff
@@ -2,35 +2,35 @@
 title: "Introduction"
 ---
 
-See the snippet below to learn how to configure your Echo SDK client.
+See the snippet below to learn how to configure your Bridge endpoint application.
 
 ```tsx
-import { Echo } from '@novu/echo';
+import { Client } from '@novu/framework';
 
-const echo = new Echo({
+const client = new Client({
```

Comment:
❓ Do we need to expose the client for now, or should we keep it for advanced use cases?

Reply:
This is dedicated docs for the client, I would imagine this makes sense

Reply:
I can update the client page to state that if explicit client config is needed, then set it up like in this and then proceed with explaining what props are available and also what the defaults are.

Then we can omit the client in all of the guides if we wish, assuming the user can find out stuff about the client from this page.